Content.Shared/Chemistry/Components/InjectorComponent.cs

```diff
@@ -81,6 +81,12 @@ public sealed partial class InjectorComponent : Component
     [AutoNetworkedField]
     [DataField]
     public InjectorToggleMode ToggleState = InjectorToggleMode.Draw;
+
+    [DataField]
+    public bool NeedHand = true;
+
+    [DataField]
+    public bool BreakOnHandChange = true;
```

Comment:
Same thing here, since I'm finding it hard to imagine why BreakOnChange would need to be false

Reply:
I made this one modifiable mainly because it appears to be quite related to the NeedHand variable. I haven't yet found a case where it would be set to false, but I don't see any reason to leave it hardcoded.

On another note, I think that I might turn couple other variables to be modifiable, particularly the one that tells how still you need to be, without it my WIP medibot change will need a nasty 0.1 second wait operator to let the bot stop moving, which currently translates to like 6 lines of yml, or 2 primitive tasks.